_includes/portfolio_grid.html

```diff
@@ -0,0 +1,90 @@
+        </div>
+      </div>
+    </div>
+    <div class="row">
+	{% for project in site.portfolio %}
+      <div class="col-md-4 col-sm-6 portfolio-item">
+        <a class="portfolio-link" data-toggle="modal" href="#p{{ forloop.index }}">
+          <div class="portfolio-hover">
+            <div class="portfolio-hover-content">
+              <i class="{{ site.data.style.portfolio-icon | default: "fas fa-plus fa-3x" }}"></i>
+            </div>
+          </div>
+          <img class="img-fluid" src="{{ project.caption.thumbnail }}" alt="">
+        </a>
+        <div class="portfolio-caption">
+          <h4>{{ project.caption.title }}</h4>
+          <p class="text-muted">{{ project.caption.subtitle }}</p>
+        </div>
+      </div>
+	{% endfor %}
+    </div>
+  </div>
+</section>
+{% else %}
+<section class="bg-light page-section" id="{{ site.data.sitetext.portfolio.section | default: "portfolio" }}">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-12 text-center">
+        <h2 class="section-heading text-uppercase">{{ site.data.sitetext.portfolio.title | markdownify | default: "Portfolio" }}</h2>
+      </div>
+      <div class="col-lg-6 text-center">
+        <h3 class="section-heading text-uppercase">🌲 Trees planted this year:</h3>
+        <h2 class="section-heading text-uppercase text-primary">{{ site.data.sitetext.portfolio.trees_planted_this_year | markdownify | default: "0" }}</h2>
+      </div>
+      <div class="col-lg-6 text-center">
+        <h3 class="section-heading text-uppercase">🌲 Total trees planted:</h3>
+        <h2 class="section-heading text-uppercase text-primary">{{ site.data.sitetext.portfolio.trees_planted_total | markdownify | default: "0" }}</h2>
+      </div>
+    </div>
+    <div class="row">
+	{% for project in site.portfolio %}
+      <div class="col-md-4 col-sm-6 portfolio-item">
+        <a class="portfolio-link" data-toggle="modal" href="#p{{ forloop.index }}">
+          <div class="portfolio-hover">
+            <div class="portfolio-hover-content">
+              <i class="{{ site.data.style.portfolio-icon | default: "fas fa-plus fa-3x" }}"></i>
+            </div>
+          </div>
+          <img class="img-fluid" src="{{ project.caption.thumbnail }}" alt="">
+        </a>
+        <div class="portfolio-caption">
+          <h4>{{ project.caption.title }}</h4>
+          <p class="text-muted">{{ project.caption.subtitle }}</p>
+        </div>
+      </div>
+	{% endfor %}
+    </div>
+  </div>
+</section>
+{% endif %}
```

Comment:
Same as previous comment.

Reply:
Thanks for pointing this out, have addressed this in commit https://github.com/SwanseaCompSci/tree-society-website/pull/6/commits/c1fc914dce23e7c9a51b0d4b32a59b4eb19e4ea1